application/src/test/java/org/thingsboard/server/edge/DeviceEdgeTest.java

```diff
@@ -555,9 +555,9 @@ public void testSendDeviceToCloud() throws Exception {
         Assert.assertTrue(edgeImitator.waitForResponses());
         Assert.assertTrue(edgeImitator.waitForMessages());
 
-        AbstractMessage latestMessage = edgeImitator.getLatestMessage();
```

Comment:
What is the reason for these changes? 
If we expect 1 response message, it should be the latest message.